ansible/roles/kubernetes-addons/templates/cluster-logging/fluentd-es-ds.yaml.j2

```diff
@@ -42,6 +42,6 @@ spec:
       - name: varlibdockercontainers
         hostPath:
           path: /var/lib/docker/containers
-      securityContext:
+    securityContext:
```

Comment:
`securityContext:` needs to be indented two more spaces in all of these, otherwise you get `found invalid field securityContext for v1.PodTemplateSpec;`